language/German/strings.po

```diff
@@ -1225,40 +1208,38 @@ msgid ""
 "\n"
 "If this is your first time using %2$s, please see the config.txt file in the "
 "docs directory for information on configuring %2$s."
-msgstr ""
+msgstr "Keine Maschinen gefunden. Bitte überprüfen Sie den Pfad für die ROMs in Ini-Datei %1$s.ini.\n\nWenn dies Ihre erste Verwendung von  %2$s ist, konsultieren Sie bitte config.txt im docs Verzeichnis zur Konfiguration von %2$s."
 
```

Comment:
Na ja, wie gesagt, es ist ja nur eine Geschmacksfrage, und wenn die Mehrheit mit "konsultieren" einverstanden ist, sei es so.

Ich hätte geschrieben "... schauen Sie bitte in die Datei config.txt im Verzeichnis "docs" zur Konfiguration von %2$s."

oder "... lesen Sie bitte die Datei ..." 

('Verzeichnis "docs"' finde ich besser als 'docs-Verzeichnis')

Jedenfalls finde ich, dass wir hier eine deutsche Übersetzung bekommen, die sich insgesamt sehen lassen kann und nicht von behelfsmäßigen Übertragungen aus dem Englischen oder sonstigen Schreibschwächen strotzt, wie man es häufig anderswo erleben muss.